sdk/test-utils/recorder/src/matcher.ts

```diff
@@ -64,9 +65,13 @@ export async function setMatcher(
       ignoreQueryOrdering: matcherBody.ignoreQueryOrdering,
     });
   }
-  const { status, bodyAsText } = await httpClient.sendRequest(request);
+
+  logger.info("Setting matcher", matcher, matcherBody);
```

Comment:
looks nice, can you also add a section in the  readme just like the regular SDKs?

Reply:
Done, let me know if you have any feedback about where else I can add logging and the messages too :)

Reply:
Good question.

service-bus SDK I think has the most dense logging if you want to draw inspirations from.

One thing to consider is to add prefixes to differentiate the messages among various paths, for example, a common prefix for all the matcher logger statements and a different prefix for all the sanitizer-related statements, etc.

If there is some calculation involved, can add a log statement there, for example, file path/name calculations, if-else blocks, etc.

Can use method names as prefixes, Example - `"[Class#methodName] ...This is a log message that says something..."`, for easier navigation from logs to code. 

Can also consider adding log statements wherever there are `await`s.